src/bag_launcher.cpp

```diff
@@ -43,6 +43,8 @@ BagLauncher::BagLauncher(ros::NodeHandle nh, BLOptions options):
     if(options.publish_name) {
         name_publisher_ = nh.advertise<bag_recorder::Rosbag>(sanitize_topic(options.name_topic), 5);
     }
+    // Set up the timer to call check_all every 0.5 secs
+    timer_ = nh.createTimer(ros::Duration(0.5), &BagLauncher::check_all, this);
```

Comment:
I think it needs to check at a rate faster than our fastest topic, with the while loop we had it set to 1000Hz, but I think 200Hz would do